projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -142,6 +146,13 @@ export const withAIAssistant = createHigherOrderComponent(
 
 		return (
 			<>
+				{ suggestionError && (
+					<ErrorNotice
+						message={ suggestionError.message }
+						onRemove={ () => setSuggestionError( null ) }
```

Comment:
Could we consider avoiding the arrow function here?

```suggestion
						onRemove={ setSuggestionError }
```

Reply:
Going to create a small `clearSuggestionError` function to be used here.

Reply:
Was not necessary since using the native notice support made me remove the error state inside the hook.